charts/incubator/notifiarr/values.yaml

```diff
@@ -0,0 +1,54 @@
+image:
+  repository: tccr.io/truecharts/notifiarr
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:913c71f5e325dd9af53c9e96f3d5d7d1b2d058a41f8253489320a4c69fbb0a44
+
+notifiarr:
+  upstreams:
+    - 172.16.0.0/16
+    - upstream1
+    - upstream2
+    - upstream3
+    - upstream4
```

Comment:
Those are at .Values.notifiarr in values.yaml but on questions you still have them under .Values.workload.main.podSpec.containers.main.env`

Also I don't get why the default values are upstream{1..4}?